modules/ROOT/pages/jackpot/index.adoc

```diff
@@ -42,19 +42,48 @@ For maven projects, this typically means placing the file into `src/main/resourc
 
 The format of the file is described xref:./HintsFileFormat.adoc[here].
 
-== Using Maven to Run Declarative Refactorings
+== Using Maven to Run Declarative Refactorings or Standard Java Hints
 
-To use run the declarative hints in a Maven project, add the tool to the build plugins in pom.xml:
-[source,java]
+To use the declarative hints or standard Java hints in a Maven project, either for verification,
+or for transformation, add the tool to the build plugins in pom.xml:
+[source,xml]
 ----
 <plugin>
     <groupId>org.apache.netbeans.modules.jackpot30</groupId>
     <artifactId>jackpot30-maven-plugin</artifactId>
-    <version>13.0</version>
+    <version>20.0</version>
 </plugin>
 ----
 
-And declare the hints in `.hint` files under `src/main/resources/META-INF/upgrade`, for example:
+Configuration can be specified like this:
+[source,xml]
+----
+<plugin>
+    <groupId>org.apache.netbeans.modules.jackpot30</groupId>
+    <artifactId>jackpot30-maven-plugin</artifactId>
+    <version>20.0</version>
+    <configuration>
+         <hint>SizeReplaceableByIsEmpty</hint>
```

Comment:
Having a list of available values somewhere would be benefitial.

Having such list in a documentation would help people understand the value of Jackpot without trying it out manually.